### DISCUSSION TITLE
Recieving JSON in Rocket 0.5.0

### DISCUSSION THREAD

**Discussion by jafioti** [rwf2/Rocket#1779] [Questions] [answered]
I'm trying to upgrade my server to 0.5.0 to get off of nightly, and I'm having an issue receiving JSON bodies. Previously I was able to specify an argument as:
```
arg: Json<Value>
```
where Json is rocket_contrib::json::Json and value is serde_json::Value. Now I get 
```
the trait bound `rocket_contrib::json::Json<serde_json::Value>: rocket::data::FromData<'_>` is not satisfied
```
How is it possible to recieve JSON in the new version? Do I need to manually implement FromData for Json<Value>? I've also tried to use rocket_contrib::json::JsonValue but got the same issue.

**Comment by jebrosen** [comment, marked as answer]
I think this is a duplicate of #1750: `rocket_contrib` [has been removed](https://github.com/SergioBenitez/Rocket/blob/v0.5-rc/CHANGELOG.md#contrib-graduation) and its functionality has been moved to `rocket` and other crates.

**Reply by jafioti** [reply]
Thanks! Sorry for the repeat question.
